Las/Core/Prediction.cs

```diff
@@ -0,0 +1,112 @@
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using Lucidtech.Las.Utils;
+
+namespace Lucidtech.Las.Core
+{
+    /// <summary>
+    /// A class that contains all the necessary information regarding a prediction performed by <see cref="ApiClient"/>.
+    /// </summary>
+    public class Prediction
+    {
+
+        /// Dictionary containing document id, consent id and model name
+        public Dictionary<string, string> Essentials { get; }
+        /// A list of the responses from a prediction
+        public List<Dictionary<string, object>> Fields { get; }
+
+        /// <summary>
+        /// Constructor of s Prediction object
+        /// </summary>
+        /// <param name="documentId"> The id of the document used in the prediction</param>
+        /// <param name="consentId"> The consent id</param>
+        /// <param name="modelName"> The name of the model used</param>
+        /// <param name="predictionResponse"> The response from prediction </param>
+        public Prediction(string documentId, string consentId, string modelName,
+            List<Dictionary<string, object>> predictionResponse)
+        {
+            Essentials = new Dictionary<string, string>()
+            {
+                {"documentId", documentId},
+                {"consentId", consentId},
+                {"modelName", modelName}
+            };
+            Fields = predictionResponse;
+        }
+
+        /// <summary>
+        /// Make the members of Prediction accessible as if it was a dictionary
+        /// </summary>
+        /// <param name="s"> a string that needs to match either one of the keys in <c> Essentials</c> or "fields" </param>
+        /// <exception cref="KeyNotFoundException"> Will throw an exception if <paramref name="s"/> is invalid</exception>
+        public object this[string s]
```

Comment:
Since this class is implemented as a wrapper around a JSON response from the api and operator[] is also defined, would it be convenient to implement functionality to easily print the whole response? I'm not sure what the best way to do this in C# is or if it is already there.